ApiTestingSolution/ApiTestingSolution.csproj

```diff
@@ -10,6 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Content Remove="C:\Users\AntonMaskalik\.nuget\packages\allure.net.commons\2.12.1\contentFiles\any\netstandard2.0\allureConfig.Template.json" />
```

Comment:
I would suggest to remove this section as it's not not applicable to keep references on local files into global and common configs.